woocommerce/woocommerce-admin
# Include orders without coupons when excluding specific coupon IDs.

Fixes #6824.

This PR fixes a bug with the Orders query when using the `coupon_excludes` parameter. Orders that didn't have coupons applied were errantly excluded from the results, causing a mismatch between the query totals (in the table summary and summary numbers up top) and the table of results.

### Screenshots

**Before:**

![Screen Shot 2021-05-19 at 14 23 59](https://user-images.githubusercontent.com/63922/118864369-267c7900-b89d-11eb-9a94-9c844718efdb.jpg)

**After:**

![Screen Shot 2021-05-19 at 14 21 25](https://user-images.githubusercontent.com/63922/118864430-36945880-b89d-11eb-8798-6c2e48371454.jpg)

### Detailed test instructions:

- Verify you have orders with and without coupons
- Go to the Orders Report (Analytics > Orders)
- Select a date range covering orders with and without coupons
- Add an advanced filter excluding a coupon
- Verify that orders without coupons show in the result set and the number of rows in the table matches the summary numbers
